main.py

```diff
@@ -0,0 +1,40 @@
+import requests
+from bs4 import BeautifulSoup
+
+url = 'https://pogoda.meta.ua/ua/Kyivska/Kyivskiy/Kyiv/'
+
+
+def temperature():
```

Comment:
Functions and methods usually starts with a verb.
```suggestion
def get_temperature():
```